diff --git a/Assets/Scenes/BuildScene's/MainScene.unity b/Assets/Scenes/BuildScene's/MainScene.unity
index 098c1a8..067f355 100644
--- a/Assets/Scenes/BuildScene's/MainScene.unity
+++ b/Assets/Scenes/BuildScene's/MainScene.unity
@@ -1181,7 +1181,7 @@ Transform:
   m_ConstrainProportionsScale: 0
   m_Children: []
   m_Father: {fileID: 0}
-  m_RootOrder: 3
+  m_RootOrder: 4
   m_LocalEulerAnglesHint: {x: 50, y: -30, z: 0}
 --- !u!1 &106336390
 GameObject:
@@ -1249,7 +1249,7 @@ Transform:
   m_ConstrainProportionsScale: 0
   m_Children: []
   m_Father: {fileID: 0}
-  m_RootOrder: 4
+  m_RootOrder: 6
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
 --- !u!114 &109331587 stripped
 MonoBehaviour:
@@ -2428,7 +2428,7 @@ Transform:
   - {fileID: 2134058171}
   - {fileID: 589057976}
   m_Father: {fileID: 0}
-  m_RootOrder: 1
+  m_RootOrder: 2
   m_LocalEulerAnglesHint: {x: 0, y: 180, z: 0}
 --- !u!1001 &358661686
 PrefabInstance:
@@ -2499,25 +2499,9 @@ PrefabInstance:
   m_Modification:
     m_TransformParent: {fileID: 0}
     m_Modifications:
-    - target: {fileID: 1089470389, guid: 52daefac7cc28f04cb02c74dc4fddc00, type: 3}
-      propertyPath: m_Materials.Array.data[0]
-      value: 
-      objectReference: {fileID: 2100000, guid: 2d1da123ce89c054fac767acebaefeda, type: 2}
-    - target: {fileID: 1200961345, guid: 52daefac7cc28f04cb02c74dc4fddc00, type: 3}
-      propertyPath: m_Materials.Array.data[0]
-      value: 
-      objectReference: {fileID: 2100000, guid: 2d1da123ce89c054fac767acebaefeda, type: 2}
-    - target: {fileID: 1247365167, guid: 52daefac7cc28f04cb02c74dc4fddc00, type: 3}
-      propertyPath: m_Materials.Array.data[0]
-      value: 
-      objectReference: {fileID: 2100000, guid: 2d1da123ce89c054fac767acebaefeda, type: 2}
-    - target: {fileID: 1899648134, guid: 52daefac7cc28f04cb02c74dc4fddc00, type: 3}
-      propertyPath: m_Materials.Array.data[0]
-      value: 
-      objectReference: {fileID: 2100000, guid: 2d1da123ce89c054fac767acebaefeda, type: 2}
     - target: {fileID: 1388430300873739755, guid: 52daefac7cc28f04cb02c74dc4fddc00, type: 3}
       propertyPath: m_RootOrder
-      value: 2
+      value: 3
       objectReference: {fileID: 0}
     - target: {fileID: 1388430300873739755, guid: 52daefac7cc28f04cb02c74dc4fddc00, type: 3}
       propertyPath: m_LocalPosition.x
@@ -2563,14 +2547,6 @@ PrefabInstance:
       propertyPath: m_Name
       value: PlayerKart
       objectReference: {fileID: 0}
-    - target: {fileID: 1388430300873739759, guid: 52daefac7cc28f04cb02c74dc4fddc00, type: 3}
-      propertyPath: speedIncrease
-      value: 2
-      objectReference: {fileID: 0}
-    - target: {fileID: 2532826891478758063, guid: 52daefac7cc28f04cb02c74dc4fddc00, type: 3}
-      propertyPath: m_Materials.Array.data[0]
-      value: 
-      objectReference: {fileID: 2100000, guid: 2d1da123ce89c054fac767acebaefeda, type: 2}
     - target: {fileID: 5671734318780633168, guid: 52daefac7cc28f04cb02c74dc4fddc00, type: 3}
       propertyPath: corrupt
       value: 
@@ -2582,7 +2558,7 @@ MonoBehaviour:
   m_CorrespondingSourceObject: {fileID: 1388430300873739759, guid: 52daefac7cc28f04cb02c74dc4fddc00, type: 3}
   m_PrefabInstance: {fileID: 362029621}
   m_PrefabAsset: {fileID: 0}
-  m_GameObject: {fileID: 0}
+  m_GameObject: {fileID: 1472384735}
   m_Enabled: 1
   m_EditorHideFlags: 0
   m_Script: {fileID: 11500000, guid: c94dd43d1aa2fb540908ee0ac26aa9e4, type: 3}
@@ -2719,7 +2695,7 @@ Transform:
   m_ConstrainProportionsScale: 0
   m_Children: []
   m_Father: {fileID: 0}
-  m_RootOrder: 6
+  m_RootOrder: 7
   m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
 --- !u!1001 &377642407
 PrefabInstance:
@@ -8444,6 +8420,11 @@ MonoBehaviour:
   m_Script: {fileID: 11500000, guid: f4688fdb7df04437aeb418b961361dc5, type: 3}
   m_Name: 
   m_EditorClassIdentifier: 
+--- !u!1 &1472384735 stripped
+GameObject:
+  m_CorrespondingSourceObject: {fileID: 1388430300873739758, guid: 52daefac7cc28f04cb02c74dc4fddc00, type: 3}
+  m_PrefabInstance: {fileID: 362029621}
+  m_PrefabAsset: {fileID: 0}
 --- !u!1001 &1473944264
 PrefabInstance:
   m_ObjectHideFlags: 0
@@ -11046,6 +11027,295 @@ Transform:
   m_CorrespondingSourceObject: {fileID: 939666790950186410, guid: ed29e4029dffdf246976b7bb8a9305cb, type: 3}
   m_PrefabInstance: {fileID: 1940537511}
   m_PrefabAsset: {fileID: 0}
+<<<<<<< HEAD
+=======
+--- !u!1001 &1945834589
+PrefabInstance:
+  m_ObjectHideFlags: 0
+  serializedVersion: 2
+  m_Modification:
+    m_TransformParent: {fileID: 0}
+    m_Modifications:
+    - target: {fileID: 6583261504931031654, guid: 6c3e2f8a68163b847bacd371741ce8fe, type: 3}
+      propertyPath: player
+      value: 
+      objectReference: {fileID: 1472384735}
+    - target: {fileID: 6996745815385279641, guid: 6c3e2f8a68163b847bacd371741ce8fe, type: 3}
+      propertyPath: m_Name
+      value: Mouth Hazard
+      objectReference: {fileID: 0}
+    - target: {fileID: 7848736159916778880, guid: 6c3e2f8a68163b847bacd371741ce8fe, type: 3}
+      propertyPath: m_RootOrder
+      value: 1
+      objectReference: {fileID: 0}
+    - target: {fileID: 7848736159916778880, guid: 6c3e2f8a68163b847bacd371741ce8fe, type: 3}
+      propertyPath: m_LocalPosition.x
+      value: -15.105751
+      objectReference: {fileID: 0}
+    - target: {fileID: 7848736159916778880, guid: 6c3e2f8a68163b847bacd371741ce8fe, type: 3}
+      propertyPath: m_LocalPosition.y
+      value: 0.3999033
+      objectReference: {fileID: 0}
+    - target: {fileID: 7848736159916778880, guid: 6c3e2f8a68163b847bacd371741ce8fe, type: 3}
+      propertyPath: m_LocalPosition.z
+      value: -33.2881
+      objectReference: {fileID: 0}
+    - target: {fileID: 7848736159916778880, guid: 6c3e2f8a68163b847bacd371741ce8fe, type: 3}
+      propertyPath: m_LocalRotation.w
+      value: 1
+      objectReference: {fileID: 0}
+    - target: {fileID: 7848736159916778880, guid: 6c3e2f8a68163b847bacd371741ce8fe, type: 3}
+      propertyPath: m_LocalRotation.x
+      value: 0
+      objectReference: {fileID: 0}
+    - target: {fileID: 7848736159916778880, guid: 6c3e2f8a68163b847bacd371741ce8fe, type: 3}
+      propertyPath: m_LocalRotation.y
+      value: 0
+      objectReference: {fileID: 0}
+    - target: {fileID: 7848736159916778880, guid: 6c3e2f8a68163b847bacd371741ce8fe, type: 3}
+      propertyPath: m_LocalRotation.z
+      value: 0
+      objectReference: {fileID: 0}
+    - target: {fileID: 7848736159916778880, guid: 6c3e2f8a68163b847bacd371741ce8fe, type: 3}
+      propertyPath: m_LocalEulerAnglesHint.x
+      value: 0
+      objectReference: {fileID: 0}
+    - target: {fileID: 7848736159916778880, guid: 6c3e2f8a68163b847bacd371741ce8fe, type: 3}
+      propertyPath: m_LocalEulerAnglesHint.y
+      value: 0
+      objectReference: {fileID: 0}
+    - target: {fileID: 7848736159916778880, guid: 6c3e2f8a68163b847bacd371741ce8fe, type: 3}
+      propertyPath: m_LocalEulerAnglesHint.z
+      value: 0
+      objectReference: {fileID: 0}
+    m_RemovedComponents: []
+  m_SourcePrefab: {fileID: 100100000, guid: 6c3e2f8a68163b847bacd371741ce8fe, type: 3}
+--- !u!1001 &1948015178
+PrefabInstance:
+  m_ObjectHideFlags: 0
+  serializedVersion: 2
+  m_Modification:
+    m_TransformParent: {fileID: 910455107}
+    m_Modifications:
+    - target: {fileID: 9180413971570856376, guid: 7f720d0f8e6ba344dbe26bb0a9d56d01, type: 3}
+      propertyPath: m_Name
+      value: WASDcontroler
+      objectReference: {fileID: 0}
+    - target: {fileID: 9180413971570856377, guid: 7f720d0f8e6ba344dbe26bb0a9d56d01, type: 3}
+      propertyPath: m_Pivot.x
+      value: 0.5
+      objectReference: {fileID: 0}
+    - target: {fileID: 9180413971570856377, guid: 7f720d0f8e6ba344dbe26bb0a9d56d01, type: 3}
+      propertyPath: m_Pivot.y
+      value: 0.5
+      objectReference: {fileID: 0}
+    - target: {fileID: 9180413971570856377, guid: 7f720d0f8e6ba344dbe26bb0a9d56d01, type: 3}
+      propertyPath: m_RootOrder
+      value: 0
+      objectReference: {fileID: 0}
+    - target: {fileID: 9180413971570856377, guid: 7f720d0f8e6ba344dbe26bb0a9d56d01, type: 3}
+      propertyPath: m_AnchorMax.x
+      value: 0.5
+      objectReference: {fileID: 0}
+    - target: {fileID: 9180413971570856377, guid: 7f720d0f8e6ba344dbe26bb0a9d56d01, type: 3}
+      propertyPath: m_AnchorMax.y
+      value: 0.5
+      objectReference: {fileID: 0}
+    - target: {fileID: 9180413971570856377, guid: 7f720d0f8e6ba344dbe26bb0a9d56d01, type: 3}
+      propertyPath: m_AnchorMin.x
+      value: 0.5
+      objectReference: {fileID: 0}
+    - target: {fileID: 9180413971570856377, guid: 7f720d0f8e6ba344dbe26bb0a9d56d01, type: 3}
+      propertyPath: m_AnchorMin.y
+      value: 0.5
+      objectReference: {fileID: 0}
+    - target: {fileID: 9180413971570856377, guid: 7f720d0f8e6ba344dbe26bb0a9d56d01, type: 3}
+      propertyPath: m_SizeDelta.x
+      value: 100
+      objectReference: {fileID: 0}
+    - target: {fileID: 9180413971570856377, guid: 7f720d0f8e6ba344dbe26bb0a9d56d01, type: 3}
+      propertyPath: m_SizeDelta.y
+      value: 100
+      objectReference: {fileID: 0}
+    - target: {fileID: 9180413971570856377, guid: 7f720d0f8e6ba344dbe26bb0a9d56d01, type: 3}
+      propertyPath: m_LocalPosition.x
+      value: 0
+      objectReference: {fileID: 0}
+    - target: {fileID: 9180413971570856377, guid: 7f720d0f8e6ba344dbe26bb0a9d56d01, type: 3}
+      propertyPath: m_LocalPosition.y
+      value: 0
+      objectReference: {fileID: 0}
+    - target: {fileID: 9180413971570856377, guid: 7f720d0f8e6ba344dbe26bb0a9d56d01, type: 3}
+      propertyPath: m_LocalPosition.z
+      value: 0
+      objectReference: {fileID: 0}
+    - target: {fileID: 9180413971570856377, guid: 7f720d0f8e6ba344dbe26bb0a9d56d01, type: 3}
+      propertyPath: m_LocalRotation.w
+      value: 1
+      objectReference: {fileID: 0}
+    - target: {fileID: 9180413971570856377, guid: 7f720d0f8e6ba344dbe26bb0a9d56d01, type: 3}
+      propertyPath: m_LocalRotation.x
+      value: -0
+      objectReference: {fileID: 0}
+    - target: {fileID: 9180413971570856377, guid: 7f720d0f8e6ba344dbe26bb0a9d56d01, type: 3}
+      propertyPath: m_LocalRotation.y
+      value: -0
+      objectReference: {fileID: 0}
+    - target: {fileID: 9180413971570856377, guid: 7f720d0f8e6ba344dbe26bb0a9d56d01, type: 3}
+      propertyPath: m_LocalRotation.z
+      value: -0
+      objectReference: {fileID: 0}
+    - target: {fileID: 9180413971570856377, guid: 7f720d0f8e6ba344dbe26bb0a9d56d01, type: 3}
+      propertyPath: m_AnchoredPosition.x
+      value: -591.5
+      objectReference: {fileID: 0}
+    - target: {fileID: 9180413971570856377, guid: 7f720d0f8e6ba344dbe26bb0a9d56d01, type: 3}
+      propertyPath: m_AnchoredPosition.y
+      value: -329.99997
+      objectReference: {fileID: 0}
+    - target: {fileID: 9180413971570856377, guid: 7f720d0f8e6ba344dbe26bb0a9d56d01, type: 3}
+      propertyPath: m_LocalEulerAnglesHint.x
+      value: 0
+      objectReference: {fileID: 0}
+    - target: {fileID: 9180413971570856377, guid: 7f720d0f8e6ba344dbe26bb0a9d56d01, type: 3}
+      propertyPath: m_LocalEulerAnglesHint.y
+      value: 0
+      objectReference: {fileID: 0}
+    - target: {fileID: 9180413971570856377, guid: 7f720d0f8e6ba344dbe26bb0a9d56d01, type: 3}
+      propertyPath: m_LocalEulerAnglesHint.z
+      value: 0
+      objectReference: {fileID: 0}
+    - target: {fileID: 9180413972021204365, guid: 7f720d0f8e6ba344dbe26bb0a9d56d01, type: 3}
+      propertyPath: m_SizeDelta.x
+      value: 75
+      objectReference: {fileID: 0}
+    - target: {fileID: 9180413972021204365, guid: 7f720d0f8e6ba344dbe26bb0a9d56d01, type: 3}
+      propertyPath: m_SizeDelta.y
+      value: 75
+      objectReference: {fileID: 0}
+    - target: {fileID: 9180413972021204365, guid: 7f720d0f8e6ba344dbe26bb0a9d56d01, type: 3}
+      propertyPath: m_AnchoredPosition.x
+      value: 215
+      objectReference: {fileID: 0}
+    - target: {fileID: 9180413972021204365, guid: 7f720d0f8e6ba344dbe26bb0a9d56d01, type: 3}
+      propertyPath: m_AnchoredPosition.y
+      value: 57
+      objectReference: {fileID: 0}
+    - target: {fileID: 9180413972051415781, guid: 7f720d0f8e6ba344dbe26bb0a9d56d01, type: 3}
+      propertyPath: m_SizeDelta.x
+      value: 75
+      objectReference: {fileID: 0}
+    - target: {fileID: 9180413972051415781, guid: 7f720d0f8e6ba344dbe26bb0a9d56d01, type: 3}
+      propertyPath: m_SizeDelta.y
+      value: 75
+      objectReference: {fileID: 0}
+    - target: {fileID: 9180413972051415781, guid: 7f720d0f8e6ba344dbe26bb0a9d56d01, type: 3}
+      propertyPath: m_AnchoredPosition.x
+      value: 140
+      objectReference: {fileID: 0}
+    - target: {fileID: 9180413972051415781, guid: 7f720d0f8e6ba344dbe26bb0a9d56d01, type: 3}
+      propertyPath: m_AnchoredPosition.y
+      value: 57
+      objectReference: {fileID: 0}
+    - target: {fileID: 9180413972412764861, guid: 7f720d0f8e6ba344dbe26bb0a9d56d01, type: 3}
+      propertyPath: m_SizeDelta.x
+      value: 75
+      objectReference: {fileID: 0}
+    - target: {fileID: 9180413972412764861, guid: 7f720d0f8e6ba344dbe26bb0a9d56d01, type: 3}
+      propertyPath: m_SizeDelta.y
+      value: 75
+      objectReference: {fileID: 0}
+    - target: {fileID: 9180413972412764861, guid: 7f720d0f8e6ba344dbe26bb0a9d56d01, type: 3}
+      propertyPath: m_AnchoredPosition.x
+      value: 140
+      objectReference: {fileID: 0}
+    - target: {fileID: 9180413972412764861, guid: 7f720d0f8e6ba344dbe26bb0a9d56d01, type: 3}
+      propertyPath: m_AnchoredPosition.y
+      value: 133
+      objectReference: {fileID: 0}
+    - target: {fileID: 9180413972463707010, guid: 7f720d0f8e6ba344dbe26bb0a9d56d01, type: 3}
+      propertyPath: m_SizeDelta.x
+      value: 75
+      objectReference: {fileID: 0}
+    - target: {fileID: 9180413972463707010, guid: 7f720d0f8e6ba344dbe26bb0a9d56d01, type: 3}
+      propertyPath: m_SizeDelta.y
+      value: 75
+      objectReference: {fileID: 0}
+    - target: {fileID: 9180413972463707010, guid: 7f720d0f8e6ba344dbe26bb0a9d56d01, type: 3}
+      propertyPath: m_AnchoredPosition.x
+      value: 215
+      objectReference: {fileID: 0}
+    - target: {fileID: 9180413972463707010, guid: 7f720d0f8e6ba344dbe26bb0a9d56d01, type: 3}
+      propertyPath: m_AnchoredPosition.y
+      value: 57
+      objectReference: {fileID: 0}
+    - target: {fileID: 9180413973022894893, guid: 7f720d0f8e6ba344dbe26bb0a9d56d01, type: 3}
+      propertyPath: m_SizeDelta.x
+      value: 75
+      objectReference: {fileID: 0}
+    - target: {fileID: 9180413973022894893, guid: 7f720d0f8e6ba344dbe26bb0a9d56d01, type: 3}
+      propertyPath: m_SizeDelta.y
+      value: 75
+      objectReference: {fileID: 0}
+    - target: {fileID: 9180413973022894893, guid: 7f720d0f8e6ba344dbe26bb0a9d56d01, type: 3}
+      propertyPath: m_AnchoredPosition.x
+      value: 140
+      objectReference: {fileID: 0}
+    - target: {fileID: 9180413973022894893, guid: 7f720d0f8e6ba344dbe26bb0a9d56d01, type: 3}
+      propertyPath: m_AnchoredPosition.y
+      value: 57
+      objectReference: {fileID: 0}
+    - target: {fileID: 9180413973059753873, guid: 7f720d0f8e6ba344dbe26bb0a9d56d01, type: 3}
+      propertyPath: m_SizeDelta.x
+      value: 75
+      objectReference: {fileID: 0}
+    - target: {fileID: 9180413973059753873, guid: 7f720d0f8e6ba344dbe26bb0a9d56d01, type: 3}
+      propertyPath: m_SizeDelta.y
+      value: 75
+      objectReference: {fileID: 0}
+    - target: {fileID: 9180413973059753873, guid: 7f720d0f8e6ba344dbe26bb0a9d56d01, type: 3}
+      propertyPath: m_AnchoredPosition.x
+      value: 140
+      objectReference: {fileID: 0}
+    - target: {fileID: 9180413973059753873, guid: 7f720d0f8e6ba344dbe26bb0a9d56d01, type: 3}
+      propertyPath: m_AnchoredPosition.y
+      value: 133
+      objectReference: {fileID: 0}
+    - target: {fileID: 9180413973161431801, guid: 7f720d0f8e6ba344dbe26bb0a9d56d01, type: 3}
+      propertyPath: m_SizeDelta.x
+      value: 75
+      objectReference: {fileID: 0}
+    - target: {fileID: 9180413973161431801, guid: 7f720d0f8e6ba344dbe26bb0a9d56d01, type: 3}
+      propertyPath: m_SizeDelta.y
+      value: 75
+      objectReference: {fileID: 0}
+    - target: {fileID: 9180413973161431801, guid: 7f720d0f8e6ba344dbe26bb0a9d56d01, type: 3}
+      propertyPath: m_AnchoredPosition.x
+      value: 65
+      objectReference: {fileID: 0}
+    - target: {fileID: 9180413973161431801, guid: 7f720d0f8e6ba344dbe26bb0a9d56d01, type: 3}
+      propertyPath: m_AnchoredPosition.y
+      value: 57
+      objectReference: {fileID: 0}
+    - target: {fileID: 9180413973268548199, guid: 7f720d0f8e6ba344dbe26bb0a9d56d01, type: 3}
+      propertyPath: m_SizeDelta.x
+      value: 75
+      objectReference: {fileID: 0}
+    - target: {fileID: 9180413973268548199, guid: 7f720d0f8e6ba344dbe26bb0a9d56d01, type: 3}
+      propertyPath: m_SizeDelta.y
+      value: 75
+      objectReference: {fileID: 0}
+    - target: {fileID: 9180413973268548199, guid: 7f720d0f8e6ba344dbe26bb0a9d56d01, type: 3}
+      propertyPath: m_AnchoredPosition.x
+      value: 65
+      objectReference: {fileID: 0}
+    - target: {fileID: 9180413973268548199, guid: 7f720d0f8e6ba344dbe26bb0a9d56d01, type: 3}
+      propertyPath: m_AnchoredPosition.y
+      value: 57
+      objectReference: {fileID: 0}
+    m_RemovedComponents: []
+  m_SourcePrefab: {fileID: 100100000, guid: 7f720d0f8e6ba344dbe26bb0a9d56d01, type: 3}
+>>>>>>> cfb88fdcb0159b082e65eed6401610b2636dc3d5
 --- !u!1001 &1968520815
 PrefabInstance:
   m_ObjectHideFlags: 0
